scripts/manage-db.py

```diff
@@ -130,22 +150,64 @@ def extract_active_data(url, dump_location='dump.sql'):
                OR releases.name = rules_scheduled_changes.base_mapping \
                OR releases.name = rules_scheduled_changes.base_whitelist \
             )" \
-        >> %s' % (mysql_default_command, database, dump_location,))
+        >> %s' % (mysql_default_command, database, dump_location))
     )
 
     popen(
         _strip_multiple_spaces('%s %s releases_history --where="releases_history.name=\'Firefox-mozilla-central-nightly-latest\' \
-        AND 1 limit 50" >> %s' % (mysql_default_command, database, dump_location,))
+        limit 50" >> %s' % (mysql_default_command, database, dump_location))
     )
 
     popen(
         _strip_multiple_spaces('%s --skip-add-drop-table --no-create-info %s releases_history --where "name = ( \
             SELECT rules.mapping \
             FROM rules \
             WHERE rules.alias=\'firefox-release\' \
-        ) LIMIT 50" >> %s' % (mysql_default_command, database, dump_location,))
+        ) LIMIT 50" >> %s' % (mysql_default_command, database, dump_location))
     )
 
+    query_release_mapping = """SELECT rules.mapping \
+            FROM rules \
+            WHERE rules.mapping IS NOT NULL \
+            UNION \
+            SELECT rules.fallbackMapping \
+            FROM rules \
+            WHERE rules.fallbackMapping IS NOT NULL \
+            UNION \
+            SELECT rules.whitelist \
+            FROM rules \
+            WHERE rules.whitelist IS NOT NULL"""
+
+    popen(_strip_multiple_spaces('''%s %s releases --where="releases.name IN (%s)" \
+        >> %s''' % (mysql_default_command, database, query_release_mapping, dump_location)))
+
+    query_release = """SELECT * \
+    FROM releases \
+    WHERE releases.name IN (%s)""" % query_release_mapping
+
+    result = trans.execute(query_release).fetchall()
+    partial_release_names = []
```

Comment:
I tried using set , however this error message kept proping up :  

mysqldump: Couldn't execute 'SELECT /*!40001 SQL_NO_CACHE */ * FROM `releases` WHERE releases.name IN (Firefox-mozilla-aurora-nightly-20160926004000, Firefox-45.0b3-build1, Firefox-48.0-build2, Thunderbird-38.4.0-build3, Firefox-mozilla-aurora-nightly-20160921004005, Thunderbird-42.0b2-build1, Firefox-49.0-build4, Firefox-oak-nightly-20160921233010, Firefox-45.0b1-build1, Firefox-mozilla-aurora-nightly-20160922004007, Firefox-mozilla-central-nightly-20160925030226, Firefox-43.0-build1, Firefox-45.0.1-build1, Firefox-43.0.4-build3, Firefox-mozilla-central-nightly-20160926030203, Firefox-50.0b1-build2, Thunderbird-45.1b1-build1, Firefox-43.0b8-build1, Firefox-44.0-build2, Firefox-45.0esr-build2, Firefox-maple-nightly-20150812100120, Firefox-49.0b2-build1, Firefox-45.1.1esr-build1, Thunderbird-38.2.0-build1, Firefox-38.0.1esr-build2, Firefox-45.3.0esr-build1, Firefox-38.3.0esr-build2, Firefox-46.0.1-build1, Firefox-49.0b10-build1, Firefox-45.0.2-build1, Firefox-48.0.2-build1, Firefox-42.0-build2, T